### ISSUE TITLE
Merchant CMS Listing Error after Shopware Update

### ISSUE THREAD

**Issue by DirkPersky** [shopwareDowntown/downtown#340]
```
[2020-10-12 13:08:07] php.CRITICAL: Uncaught Error: Too few arguments to function Shopware\Core\Content\Product\SalesChannel\Listing\ProductListingSortingRegistry::__construct(), 1 passed in /downtown.gutenberghaus.com/htdocs/var/cache/prod_hff03aa31f7bff785bea7cbae3c7643bd/ContainerNXKcdoc/srcShopware_Production_KernelProdContainer.php on line 19067 and exactly 2 expected {"exception":"[object] (ArgumentCountError(code: 0): Too few arguments to function Shopware\\Core\\Content\\Product\\SalesChannel\\Listing\\ProductListingSortingRegistry::__construct(), 1 passed in /downtown.gutenberghaus.com/htdocs/var/cache/prod_hff03aa31f7bff785bea7cbae3c7643bd/ContainerNXKcdoc/srcShopware_Production_KernelProdContainer.php on line 19067 and exactly 2 expected at /downtown.gutenberghaus.com/htdocs/vendor/shopware/core/Content/Product/SalesChannel/Listing/ProductListingSortingRegistry.php:26)"} []
```

Shopware Version: v6.3.2.0

**Comment by DirkPersky** [comment]
![screenshot-downtown gutenberghaus com-2020 10 12-15_18_12](https://user-images.githubusercontent.com/1736266/95750961-347e7180-0c9e-11eb-887a-351486e164e1.png)

**Comment by shyim** [comment]
Hey,

We use still the 6.2.1.1 on prod. I guess this https://github.com/shopwareDowntown/downtown/commit/975d889ba7d2f3d8c6957937ac6ae701790ed707 fixes your Problem (not tested).

**Comment by DirkPersky** [comment]
Jeah, 1 step forward xD
![screenshot-downtown gutenberghaus com-2020 10 13-12_56_01](https://user-images.githubusercontent.com/1736266/95852135-8c2ae480-0d53-11eb-9bfa-9bd652c2b726.png)

6.2.1.1 has some security issues, i don't think we should take that old Version.

**Comment by shyim** [comment]
I would suggest stay at the old version with the Security Plugin. I will look after my Vacation into it.

**Comment by LiaraAlis** [comment]
@shyim I think this issue is resolved since the update to Shopware 6.3.0.1.
